logger/include/logger_lib.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef LOGGER_LIB_HPP
+#define LOGGER_LIB_HPP
+
+#include <string>
+#include <memory>
+
```

Comment:
Why this should the part of this pull request

Reply:
Yes ,correct memory is not required here , removing this .